package.json

```diff
@@ -9,15 +9,15 @@
   "scripts": {
     "dev": "ts-node-dev --no-notify -P tsconfig.server.json ./server/app.ts",
     "typeorm": "ts-node -P tsconfig.server.json ./node_modules/.bin/typeorm",
```

Comment:
```suggestion
  "typeorm": "ts-node -P tsconfig.server.json ./node_modules/typeorm/cli.js",
```

Reply:
I can't verify that this works on other platforms. Just mine which is: Git Bash on Windows.